spec/requests/directory_items_controller_spec.rb

```diff
@@ -28,9 +28,10 @@
 
   context "with data" do
     before do
-      Fabricate(:evil_trout)
-      Fabricate(:walter_white)
-      Fabricate(:staged, username: 'stage_user')
+      @evil_trout = Fabricate(:evil_trout)
```

Comment:
These can be replaced with `let!(:evil_trout) { Fabricate(:evil_trout) }`

Reply:
That doesn't seem to work in this case:

> `let!` is not available from within an example (e.g. an `it` block) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc). It is only available on an example group (e.g. a `describe` or `context` block).